<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWIndow</class>
 <widget class="QMainWindow" name="MainWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Euler Square</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Montserrat SemiBold</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Choose an algorithm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEvolutionary">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Evolutionary Algorithm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHillClimb">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hill Climbing Algorithm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPSO">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Particle Swarm Optimisation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
